Wrapper for vcf2maf converter

DM2zzion opened the pull request:
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [X] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

bgruening commented on tools/vcf2maf/vcf2maf.xml:
```diff
@@ -0,0 +1,145 @@
+			<output name="output1" file="output_test1.tabular" ftype="tabular" />
+		</test>
+		<test expect_num_outputs="1">
+			<param name="input1" dbkey="dm6" value="input_test2.vcf" ftype="vcf" />
```
a test using the fasta_indexes would be nice ...

bgruening reviewed (commented):
Very cool! Thanks @DM2zzion this is high quality! Do you need help with the failing test?

bgruening commented:
> fails because it cannot write the new file $input_name.split.vcf. I suppose on the test system, the tool does not have write access in that case?

Yes the input file directory is read-only on many systems. And everything should happen in a working directory - also for security reasons.

Please try symlinking the input file into the CWD. Something along those lines:

`ln -s '$input' ./foo.vcf && vcf2mat ./foo.vcf ...`

DM2zzion commented:
> > fails because it cannot write the new file $input_name.split.vcf. I suppose on the test system, the tool does not have write access in that case?
> 
> Yes the input file directory is read-only on many systems. And everything should happen in a working directory - also for security reasons.
> 
> Please try symlinking the input file into the CWD. Something along those lines:
> 
> `ln -s '$input' ./foo.vcf && vcf2mat ./foo.vcf ...`

Thanks for your help! With the last commit I introduced symlinks for the input and reference fasta. Furthermore, I added a test case for fasta_indexes as requested by you. I think the wrapper is ready now :)

bgruening reviewed (approved):
Cool, very nice! Thanks a lot @DM2zzion
